docker-compose-developer.yml

```diff
@@ -29,6 +30,8 @@ services:
       context: ./grafana-plugin
       dockerfile: Dockerfile.dev
       labels: *oncall-labels
+    environment:
+      ONCALL_API_URL: http://host.docker.internal:8080
```

Comment:
just checking if it should be necessarily forwarded through the host network (because of the browser) or can be just http://engine:8080?

Reply:
I tested this locally. It complains because our container names in `docker-compose-developer.yml` use underscores instead of dashes (eg. `oncall_engine`). This isn't RFC 1738 compliant and the plugin-proxy complains. I propose we follow up in a future PR and make this enhancement.